src/main/java/net/minecraftforge/fluids/FluidRegistry.java

```diff
@@ -356,6 +356,21 @@ public static String getDefaultFluidName(Fluid key)
         return name;
     }
 
+    @Nullable
+    public static String getModId(@Nullable FluidStack fluidStack)
```

Comment:
Ideally `FluidRegistry` is going to be replaced with a real registry in https://github.com/MinecraftForge/MinecraftForge/pull/4098 so this method is only temporary.